.appveyor.yml

```diff
@@ -7,6 +7,6 @@ build_script:
 - cmd: >-
     mkdir build &&
     cd build &&
-    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_CXX_FLAGS="/W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
-    cmake --build . --target install &&
+    cmake c:\projects\source -DKokkos_ENABLE_TESTS=ON -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_CXX_FLAGS="/O2 /W0 /EHsc" -DKokkos_ENABLE_DEPRECATED_CODE_4=ON &&
+    cmake --build . --target install --config Release &&
```

Comment:
This builds the Release config (and we configured with the RelWithDebInfo build type), but on the next line we are only running the Debug config
I'm not really used to multi-config generators but this seems weird. We should be consistent.

Also any idea what generator we are actually using? This may be a red herring unless we are actually using a multi-config generator. E.g. Maybe we should explicitly put a multiconfig generator `-G "Visual Studio 17 2022"`

Reply:
I found [this](https://stackoverflow.com/questions/19024259/how-to-change-the-build-type-to-release-mode-in-cmake) stack overflow answer that mentions that by default with no `--config` flag to `--build`, it will build `Debug` by default, which is likely how this worked before.